src/ocamlorg_frontend/pages/learn.eml

```diff
@@ -8,12 +8,12 @@ Learn_layout.single_column_layout
 ~canonical:Url.learn
 ~active_top_nav_item:Header.Learn
 ~current:Overview @@
-  <div class="w-full border-b bg-[#DDEAFC] md:bg-[length:100%_100%] md:bg-transparent md:bg-[url('/img/learn/hero-background.svg')]">
```

Comment:
background color cannot be removed for the small screens, see the mobile design, there's a solid color background both in light mode and in dark mode